### PR TITLE
resin information disclosure

### PR DESCRIPTION
**Pull request by pikpikcu**
```
                       __     _ 
     ____  __  _______/ /__  (_)
    / __ \/ / / / ___/ / _ \/ / 
   / / / / /_/ / /__/ /  __/ /  
  /_/ /_/\__,_/\___/_/\___/_/   v2.1									  

		projectdiscovery.io

[WRN] Use with caution. You are responsible for your actions
[WRN] Developers assume no liability and are not responsible for any misuse or damage.
[INF] [Caucho-Resin-Information-Disclosure] Loaded template Caucho Resin Information Disclosure (@pikpikcu) [info]
[Caucho-Resin-Information-Disclosure] [http] https://REDACTED/resin-doc/viewfile/?file=/WEB-INF/resin-web.xml
[Caucho-Resin-Information-Disclosure] [http] https://REDACTED/%20../web-inf/web.xml
[Caucho-Resin-Information-Disclosure] [http] https://REDACTED/resin-doc/viewfile/?file=/WEB-INF/resin-web.xml
[Caucho-Resin-Information-Disclosure] [http] https://REDACTED/resin-doc/viewfile/?file=/WEB-INF/resin-web.xml

```

### CODE REVIEW

File: vulnerabilities/other/caucho-resin-info-disclosure.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    matchers-condition: and
+    matchers:
+      - type: status
+        status:
+          - 200
+      - type: word
+        words:
+          - "WEB-INF"
+        part: body
```
Comment (toufik-airane) [inline vulnerabilities/other/caucho-resin-info-disclosure.yaml:23]:
```suggestion
    matchers:
      - type: word
        words:
          - "<web-app"
          - "</web-app>"
        part: body
        condition: and
```

**Review by toufik-airane — CHANGES_REQUESTED** [review]
Dear @pikpikcu,

Thank you for your contribution.

After discussion, we've figured out that the open and close tag could be more appropriate to avoid false positives.

What do you think about the proposed changes?

Thank you.

**Comment by toufik-airane** [comment]
Dear @pikpikcu,

Just a little reminder. :)
I think we can agree that the use of the new wording could decrease false positives.

Thank you for your answer.

**Comment by toufik-airane** [comment]
Dear @ehsandeep,

Do you think we can merge this one with the changes?

Thanks.

**Review by ehsandeep — APPROVED** [review]

**Comment by ehsandeep** [comment]
made some changes, looks good to me @toufik-airane.

**Pull request merged**